app/src/lib/stores/app-store.ts

```diff
@@ -1809,6 +1801,7 @@ export class AppStore extends TypedBaseStore<IAppState> {
         this.statsStore.recordPartialCommit()
       }
 
+      const { trailers } = context
       if (trailers != null && trailers.some(isCoAuthoredByTrailer)) {
```

Comment:
I know this isn't part of your change (in fact I think I wrote this) but can we make this `!== undefined` to be clear about the fact that trailers can't be `null`, only `undefined` or an array?
```suggestion
      if (trailers !== undefined && trailers.some(isCoAuthoredByTrailer)) {
```